fe/fe-core/src/main/java/com/starrocks/sql/optimizer/ChildPropertyDeriver.java

```diff
@@ -425,6 +425,10 @@ private void tryReplicatedHashJoin(ExpressionContext context,
      * */
     private void tryBucketShuffle(PhysicalHashJoinOperator node, HashDistributionSpec leftShuffleDistribution,
                                   HashDistributionSpec rightShuffleDistribution) {
+        if (ConnectContext.get().getSessionVariable().isDisableColocateJoin()) {
```

Comment:
so we can disable bucket shuffle and disable colocate join at the same time. to align hive plan and native plan, we need to disable both.